modules/imap/js_modules/utils/handleNexterDateAction.js

```diff
@@ -0,0 +1,67 @@
+function handleNexterDateAction(e) {
+    let isReloading = false;
+
+    let reload_and_redirect = async function () {
+        if (isReloading) { return; }
+        isReloading = true;
+
+        try {
+            Hm_Folders.reload_folders(true);
+            let path = hm_list_parent() ? hm_list_parent() : hm_list_path();
```

Comment:
I'm sure you won't see the values: ` hm_list_parent() , hm_list_path()` referenced anywhere because route parameters are handled by https://github.com/cypht-org/cypht/blob/master/modules/core/navigation/utils.js

Reply:
I just added some changes and use the `showRoutingToast();` and `hideRoutingToast();` method to show and hide the redirect message. please review